tests/src/python/providertestbase.py

```diff
@@ -352,7 +377,8 @@ def testUniqueStringsMatching(self):
         self.assertEqual(len(result), 2)
         self.assertTrue(result.issubset(set(['Pear', 'Orange', 'Apple'])))
 
-        assert set([u'Apple', u'Honey', u'Orange', u'Pear', NULL]) == set(self.source.uniqueValues(field_index)), 'Got {}'.format(set(self.source.uniqueValues(field_index)))
+        assert set([u'Apple', u'Honey', u'Orange', u'Pear', NULL]) == set(
```

Comment:
Since you are on it, any chance to get a `self.assertEqual` here too?

Reply:
I've already moved on... this is a backport :P

Reply:
Let's merge if travis is happy then